### PR TITLE
Updating trademarks for Nano products

### PR DESCRIPTION
**Pull request by jhansson-ard**
## What This PR Changes
- Updating trademarks and formatting for Nano products

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/03.nano/boards/nano-every/datasheet/datasheet.md [hunk 1]
```diff
@@ -124,19 +124,19 @@ The board can be powered via USB connector, V<sub>IN</sub> or V<sub>USB</sub> pi
 
 ## Board Operation
 ### Getting Started - IDE 
-If you want to program your Arduino Nano 33 BLE while offline you need to install the Arduino Desktop IDE [1] To connect the Arduino Nano 33 BLE to your computer, you’ll need a Micro-B USB cable. This also provides power to the board, as indicated by the LED.
+If you want to program your Nano ESP32 while offline you need to install the Arduino Desktop IDE [1] To connect the Arduino Nano 33 BLE to your computer, you’ll need a Micro-B USB cable. This also provides power to the board, as indicated by the LED.
```
Comment (jacobhylen) [inline content/hardware/03.nano/boards/nano-every/datasheet/datasheet.md:127]:
```suggestion
If you want to program your Nano Every while offline you need to install the Arduino Desktop IDE [1] To connect the Arduino Nano Every to your computer, you’ll need a Micro-B USB cable. This also provides power to the board, as indicated by the LED.

```

File: content/hardware/03.nano/boards/nano-rp2040-connect/datasheet/datasheet.md [hunk 2]
```diff
@@ -159,49 +159,50 @@ The Arduino® Nano RP2040 Connect can be adapted to a wide range of use cases th
...
 
 ### Cryptography
 The ATECC608A Cryptographic IC (U4) provides secure boot capabilities alongside SHA and AES-128 encryption/decryption support for security in Smart Home and Industrial IoT (IIoT) applications. Additionally, a random number generator is also available for use by the RP2040. 
 
 ### Microphone
-The MP34DT06J microphone is connected via a PDM interface to the RP2040. The digital MEMS microphone is omnidirectional and operate via a capacitive sensing element with a high (64 dB) signal to noise ratio. The sensing element, capable of detecting acoustic waves, is manufactured using a specialized silicon micromachining process dedicated to produce audio sensors.
+The MP34DT06J microphone is connected via a PDM interface to the Nano RP2040 Connect. The digital MEMS microphone is omnidirectional and operate via a capacitive sensing element with a high (64 dB) signal to noise ratio. The sensing element, capable of detecting acoustic waves, is manufactured using a specialized silicon micromachining process dedicated to produce audio sensors.
```
Comment (jacobhylen) [inline content/hardware/03.nano/boards/nano-rp2040-connect/datasheet/datasheet.md:179]:
```suggestion
The MP34DT06J microphone is connected via a PDM interface to the RP2040. The digital MEMS microphone is omnidirectional and operate via a capacitive sensing element with a high (64 dB) signal to noise ratio. The sensing element, capable of detecting acoustic waves, is manufactured using a specialized silicon micromachining process dedicated to produce audio sensors.

```
In this case it's actually referring to the MCU, not the entire board, so just RP2040 is correct

File: content/hardware/03.nano/boards/nano-rp2040-connect/datasheet/datasheet.md [hunk 3]
```diff
@@ -159,49 +159,50 @@ The Arduino® Nano RP2040 Connect can be adapted to a wide range of use cases th
...
 The Nina W102 module also includes a dual core Xtensa LX6 CPU that can also be programmed independently of the RP2040 through the pads under the board using SWD. 
 
 ### 6-Axis IMU
 It is possible to obtain 3D gyroscope and 3D accelerometer data from the LSM6DSOX 6-axis IMU (U9). In addition to providing such data, it is also possible to do machine learning on the IMU for gesture detection.
 
 ### External Memory
-The RP2040 (U1) has access to an additional 16 MB of flash memory via a QSPI interface. The execute-in-place (XIP) feature of the RP2040 allows external flash memory to be addressed and accessed by the system as though it were internal memory,  without first copying the code to internal memory.
+The RP2040 (U1) has access to an additional 16 MB of flash memory via a QSPI interface. The execute-in-place (XIP) feature of the Nano RP2040 Connect allows external flash memory to be addressed and accessed by the system as though it were internal memory,  without first copying the code to internal memory.
```
Comment (jacobhylen) [inline content/hardware/03.nano/boards/nano-rp2040-connect/datasheet/datasheet.md:173]:
```suggestion
The RP2040 (U1) has access to an additional 16 MB of flash memory via a QSPI interface. The execute-in-place (XIP) feature of the RP2040 allows external flash memory to be addressed and accessed by the system as though it were internal memory,  without first copying the code to internal memory.

```

File: content/hardware/03.nano/boards/nano-rp2040-connect/datasheet/datasheet.md [hunk 4]
```diff
@@ -159,49 +159,50 @@ The Arduino® Nano RP2040 Connect can be adapted to a wide range of use cases th
...
 
 ### RGB LED
 The RGB LED (DL3) is a common anode LED that is connected to the Nina W102 module. The LED are off when the digital state is HIGH and on when the digital state is LOW.
 
 ### Power Tree
 ![Power Tree of Arduino Nano RP2040 Connect Topology](assets/nanoRP2040PowerTree.png)
 
-The Arduino Nano RP2040 Connect can be powered by either the Micro USB port (J1) or alternatively via VIN on JP2. An onboard buck converter provides 3V3 to the RP2040 microcontroller and all other peripherals. Additionally, the RP2040 also has an internal 1V8 regulator.
+The Nano RP2040 Connect can be powered by either the Micro USB port (J1) or alternatively via VIN on JP2. An onboard buck converter provides 3V3 to the RP2040 microcontroller and all other peripherals. Additionally, the Nano RP2040 Connect also has an internal 1V8 regulator.
```
Comment (jacobhylen) [inline content/hardware/03.nano/boards/nano-rp2040-connect/datasheet/datasheet.md:187]:
```suggestion
The Nano RP2040 Connect can be powered by either the Micro USB port (J1) or alternatively via VIN on JP2. An onboard buck converter provides 3V3 to the RP2040 microcontroller and all other peripherals. Additionally, the RP2040 also has an internal 1V8 regulator.

```

**Review by jacobhylen — CHANGES_REQUESTED** [review]
Hej @jhansson-ard 
Leaving a few comments here too :)

File: content/hardware/03.nano/boards/nano-rp2040-connect/datasheet/datasheet.md (hunk 3 above)
Comment (jhansson-ard) [reply content/hardware/03.nano/boards/nano-rp2040-connect/datasheet/datasheet.md:173]: Thanks! I was unsure of this one! ⭐

File: content/hardware/03.nano/boards/nano-every/datasheet/datasheet.md [hunk 5]
```diff
@@ -124,19 +124,20 @@ The board can be powered via USB connector, V<sub>IN</sub> or V<sub>USB</sub> pi
 
 ## Board Operation
 ### Getting Started - IDE 
-If you want to program your Arduino Nano 33 BLE while offline you need to install the Arduino Desktop IDE [1] To connect the Arduino Nano 33 BLE to your computer, you’ll need a Micro-B USB cable. This also provides power to the board, as indicated by the LED.
+If you want to program your Nano Every while offline you need to install the Arduino Desktop IDE [1] To connect the Arduino Nano Every to your computer, you’ll need a Micro-B USB cable. This also provides power to the board, as indicated by the LED.
```
Comment (jhansson-ard) [inline content/hardware/03.nano/boards/nano-every/datasheet/datasheet.md:127]:
```suggestion
If you want to program your Nano Every while offline you need to install the Arduino Desktop IDE [1] To connect the Nano Every to your computer, you’ll need a Micro-B USB cable. This also provides power to the board, as indicated by the LED.

```

**Comment by jhansson-ard** [comment]
Implemented your changes @jacobhylen, thanks for the review! ⭐

**Review by jacobhylen — APPROVED** [review]

**Pull request merged**